stage1/init/container.go

```diff
@@ -79,12 +79,35 @@ func LoadContainer(root string) (*Container, error) {
 	return c, nil
 }
 
+// quoteExec returns an array of quoted strings appropriate for systemd execStart usage
+func quoteExec(exec []string) string {
+	if len(exec) == 0 {
+		// existing callers prefix {"/diagexec", "/app/root"} so this shouldn't occur.
+		panic("empty exec")
+	}
+
+	var qexec []string
+	qexec = append(qexec, exec[0])
+	// FIXME(vc): systemd gets angry if qexec[0] is quoted
+	// https://bugs.freedesktop.org/show_bug.cgi?id=86171
+
+	if len(exec) > 1 {
+		for _, arg := range exec[1:] {
+			escArg := strings.Replace(arg, "\"", "\\\"", -1)
```

Comment:
Yes, test case coming, and the raw strings are definitely clearer.  My C habits are very deeply rooted...
Also, in thinking about the test cases I realized the escaping is missing something; escaping \'s in the words, they need to be stuffed.